docs/sources/mimir/operators-guide/architecture/deployment-modes/index.md

```diff
@@ -43,3 +43,18 @@ If you are interested in deploying Grafana Mimir in microservices mode, we recom
 [//]: # "Diagram source at https://docs.google.com/presentation/d/1LemaTVqa4Lf_tpql060vVoDGXrthp-Pie_SQL7qwHjc/edit#slide=id.g11658e7e4c6_1_53"
 
 ![Mimir's microservices mode](microservices-mode.svg)
+
+## Read-Write mode
+
+> **Warning:**
+> Read-Write deployment mode is currently considered experimental and not ready for production use.
```

Comment:
[nit] I would smooth it. We're already running it in production at a small scale ;) 

```suggestion
> Read-Write deployment mode is currently considered experimental.
```